views/student-orgs/list.js

```diff
@@ -181,28 +156,18 @@ export class StudentOrgsView extends React.Component {
     }
 
     if (!size(this.state.orgs)) {
-      return (
-        <View style={{
-          flex: 1,
-          justifyContent: 'center',
-          alignItems: 'center',
-          backgroundColor: '#ffffff',
-        }}>
-          <Text>
-            No organizations found.
-          </Text>
-        </View>
-      )
+      return <NoticeView text='No organizations found.' />
     }
 
     return (
-      <AlphabetListView
-        contentContainerStyle={styles.listView}
+      <StyledAlphabetListView
         data={this.state.orgs}
         cell={this.renderRow}
-        sectionHeader={this.renderHeader}
-        sectionHeaderHeight={28}
-        cellHeight={53}
+        // just setting cellHeight sends the wrong values on iOS.
+        cellHeight={rowHeight + (Platform.OS === 'ios' ? (11/12 * StyleSheet.hairlineWidth) : 0)}
+        sectionHeader={this.renderSectionHeader}
+        sectionHeaderHeight={headerHeight}
+        renderSeparator={this.renderSeparator}
```

Comment:
Turns out that you can use renderSeparator on AlphabetListView, too. That simplified some code!